src/main.rs

```diff
@@ -26,12 +26,19 @@ async fn main() {
 
     let state_filter = warp::any().map(move || pool.clone());
 
-    let routes = warp::any()
+    let account_routes = warp::path("account")
```

Comment:
`/account` seems kinda vague name-wise, how about something like `/create`, `/claim` or `/signup`

Reply:
That was just a quick path I thought of merely for testing, `/signup` sounds good